src/app/screens/Home/index.tsx

```diff
@@ -94,6 +100,36 @@ function Home() {
     }
   }
 
+  async function onTabChangeHandler(index: number) {
+    if (index === 1) {
+      await loadInvoices();
+    }
+  }
+
+  async function loadInvoices() {
+    // incoives have already been loaded
+    if (invoiceTransactions) return invoiceTransactions;
+
+    setLoadingInvoices(true);
+    const result = await api.getInvoices();
+
+    const invoices: Transaction[] = result.invoices
+      .filter((invoice) => invoice.settled)
```

Comment:
we could at some point add this to the action. thinking of pagination we might need arguments there anyway.

```
api.getInvoices({settled: true});

Reply:
Done